client/sidebar/header/EditStatusModal.tsx

```diff
@@ -49,6 +49,25 @@ const EditStatusModal = ({ onClose, userStatus, userStatusText }: EditStatusModa
 		onClose();
 	}, [dispatchToastMessage, statusType, statusText, setUserStatus, onClose, t]);
 
+	// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
+	const handleEmojiPicker = (event: any) => {
+		const className = document.getElementsByClassName('custome_emoji_react')[0];
+		const inputField = className?.childNodes[0];
```

Comment:
Use useref hook insted of document.getElement..

Reply:
resolved in the next [commit ] in this PR.(https://github.com/RocketChat/Rocket.Chat/pull/24967/commits/19b631ba0396757c9608c91406f49602d0596b98)
Thank you very much @nishant23122000